projects/akita-ng-fire/src/lib/auth/auth.service.ts

```diff
@@ -283,12 +280,17 @@ export class FireAuthService<S extends FireAuthState> {
       } else if (this.onSignin) {
         await this.onSignin(cred);
```

Comment:
If somebody sign-in with a new user, he won't have the onSignin callback called, is it expected?